common/rfb/CSecurityTLS.cxx

```diff
@@ -56,6 +56,12 @@ StringParameter CSecurityTLS::X509CA("X509CA", "X509 CA certificate",
 StringParameter CSecurityTLS::X509CRL("X509CRL", "X509 CRL file",
                                      configdirfn("x509_crl.pem"),
                                      ConfViewer);
+StringParameter CSecurityTLS::X509CERT("X509CERT", "X509 client certificate",
+                                     configdirfn("x509_cert.pem"),
```

Comment:
The config directory is also shared with the server, so I'm thinking we might need a more unique default name to avoid confusion. Perhaps `x509_client_cert.pem`?